.package-version

```diff
@@ -1 +1 @@
-8.14.0
\ No newline at end of file
+8.15.0
```

Comment:
The automation should do this tomorrow, https://github.com/elastic/elastic-agent/pull/4768

At which point this just becomes removing the skipped test.

Reply:
Automation update is in https://github.com/elastic/elastic-agent/pull/4773